web/html/core-html-elements/style-element.md

```diff
@@ -0,0 +1,118 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+  - basic
+
+type: normal
+
+category: must-know
+
+standards:
+  web.syntax-html.0: 10
+  web.markup-text.0: 10
+
+aspects:
+  - introduction
+
+links:
+  - '[style element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style){documentation}'
+
+---
+# Style element
+
+---
+## Content
+
+The HTML `style` element is used to style the content of an html document. Also, there is a `style` attribute used to achieve the same thing, but per element.
+
+Example 1:
+```html
+<p style="color: red"> 
+  Color of this text is red.
+</p>
+```
+Output:
+
+![image-output](https://img.enkipro.com/b199c00475956669e57f76ba58a0f654.png)
+
+The above example, colors only that single paragraph element. 
+
+If we would like to color all paragraphs withing the whole HTML document, we would add the `style` element at the top of the document.
+However, if we add a style attribute within any `<p>` element, it would override our top level style element.
```

Comment:
Maybe use "to" instead of "within"? We are adding an attribute to an element, not inside of it.

Reply:
but we are coloring all paragraphs located `within` the HTML file, not `to` the HTML file. For it to be `to`, the whole paragraph has to be rewritten. Should I rewrite it?